orc8r/cloud/go/sqorc/error_checker.go

```diff
@@ -0,0 +1,40 @@
+package sqorc
```

Comment:
it would be nice to have some tests for this file, since it's a library code

Reply:
done